.changeset/two-elephants-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-airbrake': minor
+---
+
+API connectivity has added, but currently will only work by running it standalone on a browser with CORS disabled.
```

Comment:
Why is this?

Reply:
More explanation in the PR description, but essentially this PR was made before the backend was written and until the Backend is merged it won't really work out of the box due to the CORS policies.